mantidproject/mantid
# Use xUnit output from CTest for Jenkins

**Description of work.**

xUnit reports are written to a Testing directory of the build by ctest, which can be parsed by Jenkins. This enables the output catch errors such as failures in Python tests and segfaults before the C++ test
suites run.

**To test:**

Code review.
Check the builds pass.

Locally running ctest will not produce the xml output by default. Adding `-T Test` to the `ctest` call should produce the output in the `build/Testing` directory.

*There is no associated issue.*

*This does not require release notes* because **changes for developer testing**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
